src/App.tsx

```diff
@@ -1,8 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
+import 'bootswatch/dist/lux/bootstrap.min.css';
+import { GoodList } from './components/GoodsList/GoodList';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
 
-export default App;
+  const url = 'https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json';
+
+  const downloadGoodsList = async (): Promise<Good[]> => {
+    setIsLoading(true);
```

Comment:
downloadGoodsList - move outside of the component and setIsLoading in each function
or move fetch logic to separate function outside, but leave here helper for setIsLoading